### ISSUE TITLE
Errors from post deployment hooks are swallowed and execution continues

### ISSUE THREAD

**Issue by marktani** [prismagraphql/prisma#2362]
# Bug Report

**Current behavior**

If a post deployment hook returns with error, the errors is swallowed and the next deployment hook is executed.

**Reproduction**

Have this `.graphqlconfig.yml`:

```yml
projects:
  app:
    schemaPath: "src/schema.graphql"
    extensions:
      endpoints:
        default: "http://localhost:4000"
  prisma:
    schemaPath: "src/generated/prisma.graphqls"
    extensions:
      prisma: ../prisma/prisma.yml
      prepare-binding:
        output: src/generated/prisma.ts
        generator: prisma-ts
```

and this `prisma.yml`

```yml
endpoint: ${env:PRISMA_ENDPOINT}
secret: ${env:PRISMA_SECRET}
datamodel:
  - types.graphqls
hooks:
  post-deploy:
    - graphql get-schema --project prisma
    - graphql prepare
```

Run `prisma deploy`. The error for `graphql get-schema` is swallowed, and `graphql prepare` is executed.

Note, when running `graphql get-schema` manually, an error is returned:

```
→ graphql get-schema --project prisma
✖ Unsupported schema file extention. Only ".graphql" and ".json" are supported
```

Also note https://github.com/graphql-cli/graphql-cli/issues/286 which might be related to the continuation of execution.

**Expected behavior?**

* Errors from post deployment hooks are displayed OR introduce `--verbose` flag for `prisma deploy` to do so. I prefer the former.
* Execution should stop if a command returns an error code.

**Comment by marktani** [comment]
As discussed in https://github.com/prismagraphql/prisma/issues/2560, it seems that a _missing_ `graphql` binary is also not detected.

**Comment by divyenduz** [comment]
Addressed https://github.com/prismagraphql/prisma/pull/2562 and https://github.com/graphql-cli/graphql-cli/pull/313

**Comment by divyenduz** [comment]
🎉This is released in prisma cli [version 1.9](https://github.com/prismagraphql/prisma/releases/tag/1.9). 🎉

Please download using `npm install -g prisma`
